tests/testthat/test-getFileName.R

```diff
@@ -4,6 +4,8 @@ test_that("getFileNameBatch returns the expected results", {
   cmdArgs2 <- c("--arg1", "someThing", "-e", 'timber::axecute(file="/asdf/programs/CopyOftest_demo.R",log_name="CopyOftest_demo.R.log",log_path="/adr/PDEV/emiller9/pharma/test_compound/mi_test/dbr_tidytlg/re_test/output")')
```

Comment:
@elimillera the log_path argument in cmdArgs2 is using the J&J file structure. Can we change this argument to be more general?